pkg/meta/model/job.go

```diff
@@ -897,6 +885,13 @@ type SubJob struct {
 	UseCloud    bool            `json:"use_cloud"`
 }
 
+// FillArgs fills args for subjob.
+func (sub *SubJob) FillArgs(args JobArgs, version JobVersion) {
+	job := &Job{Version: version}
+	job.FillArgs(args)
+	sub.Args = job.Args
+}
```

Comment:
it's not a generic method, prefer we abstract a more specific MergeAddIndexSubJobs